modules/sbo-unbinding-workloads-from-a-backing-service.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * /applications/connecting_applications_to_services/binding-workloads-using-sbo.adoc
+
+[id="sbo-unbinding-workloads-from-a-backing-service_{context}"]
+= Unbinding workloads from a backing service
+
+You can unbind a workload from a backing service by using the `oc` tool.
+
+. To unbind a workload from a backing service, delete the `ServiceBinding` custom resource (CR) linked to it :
+
+[source,terminal]
+----
+$ oc delete ServiceBinding <.metadata.name>
+----
+
+.Example
+[source,yaml]
```

Comment:
```suggestion
[source,terminal]
```

Reply:
Phew! I guess I was too exhausted by the end of the day. How can I miss this?